extensions/cache/src/main/resources/benchmark-redis.xml

```diff
@@ -0,0 +1,69 @@
+<benchmark xmlns="urn:radargun:benchmark:3.0">
+
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}"/>
+
+   <clusters>
+      <cluster size="8">
+         <group name="server" size="6" />
+         <group name="client" size="2" />
+      </cluster>
+   </clusters>
+
+   <configurations>
+      <config name="redis">
+         <setup group="server" plugin="redis32">
+            <server xmlns="urn:radargun:plugins:redis32:3.0" file="${env.RADARGUN_DISTRO_PATH}/plugins/redis32/conf/redis.conf"
+                    distribution-dir="/tmp/redisserver${slave.index}/" distribution-zip="${env.REDIS_DISTRO_ZIP_PATH}/redisdistro.zip" port="#{7000 + ${slave.index}}">
```

Comment:
Why does it need to be recreated? I thought about setting just distribution-dir (no unzip). Benefit is reduced unpack time & disk space. But nothing critical at this point.